### PR TITLE
Generate preview URLs using Netlify

### PR DESCRIPTION
**Pull request by dannymidnight**
Render is costing a lot for what it does. 

This PR adds an alternative preview URL solution by instead generating a static site and using Netlify for managing previews.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ def render_attribute_content(attribute)
     render(partial: "quick_reference/#{attribute}", formats: [:md]).to_json.html_safe
   end
 
-  def logo_image_url
+  def logo_image_path
```
Comment (mbelton-buildkite) [inline app/helpers/application_helper.rb:46]: Just out of curiosity: Why did these need to be updated?

**Comment by mbelton-buildkite** [comment]
Looks sensible to me! Can we see it work or does this need to be merged first?

**Comment by dannymidnight** [comment]
Preview URL: https://false--bk-docs-preview.netlify.app/

**Comment by dannymidnight** [comment]
Preview URL: https://false--bk-docs-preview.netlify.app/

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (dannymidnight) [reply app/helpers/application_helper.rb:46]: The `_url` suffix in rails outputs absolute URLs and unfortunately when scraping a local version of the site it meant the host was being output as localhost:3000. By using relative URLs I was able to output compatible markup.

**Comment by dannymidnight** [comment]
I've made a few tweaks to the main pipeline so going to re-open another PR to see if it all works as expected...

**Pull request merged**